coalib/results/result_actions/ShowPatchAction.py

```diff
@@ -125,11 +127,16 @@ def apply(self,
             if tuple(current_file) != tuple(new_file):
                 print_beautified_diff(difflib.unified_diff(current_file,
                                                            new_file,
-                                                           fromfile=filename,
+                                                           fromfile=(
```

Comment:
This change can be avoided, and lower complexity, by changing filename to `/dev/null` if `create` before the function call.